### PR TITLE
Adds butts

### PR DESCRIPTION
**Pull request by tigercat2000**
Whoo more NanoUI shit

Changes this time:
 - the uiIcon16 stuff has been ripped out and replaced completely by FontAwesome (report missing icons)
 - Technical detail: Uses the font-awesome fixed-width class because nano links are shit
 - A few icons have been replaced with not-quite-the-same icons
  - Radiation is a generic circle-alert
  - Mail open/closed is now represented by envelope; Does not have open and closed icons, instead, changes color.
  - Signal-diag has been removed, replaced by the standard `signal`

Fire alarm UI:
full
With colors for each alert level (tell me if these are shitty)
 - Green: blarg
 - Blue: blarg
 - Red: blarg
 - Gamma: blarg
 - Epsilon: blarg
 - Delta: blarg

Other showcases:
wat
mrowl

### CODE REVIEW

**Comment by tigercat2000** [comment]
Oh, and an image of the PDA alt style for example (they are normally white but whatever)
wat

**Comment by Regen1** [comment]
Misleading title

Tigr pls

**Comment by Tastyfish** [comment]
Former goonstation players will be disappointed by misleading title

**Comment by Tastyfish** [comment]
Also I guess now's the phase where we slowly replace icons with more appropriate ones.

**Comment by Tastyfish** [comment]
Also, there is [a disk icon](http://fortawesome.github.io/Font-Awesome/icon/floppy-o/).

**Comment by tigercat2000** [comment]
ah
didn't think of the word `

**Comment by KasparoVy** [comment]
Nice clickbait lol

**Comment by TheDZD** [comment]
Lack of buttbots.
Lack of butt removal surgery.
Lack of fun
:+1:

**Comment by Earthdivine** [comment]
:-1: Butt bots when.

**Comment by Plotron** [comment]
floppy > HDD

**Comment by tigercat2000** [comment]
>taking 5 seconds with a search and replace

no thanks

**Comment by tigercat2000** [comment]
mrowl

**Comment by tigercat2000** [comment]
Update: Made a fire alarm UI, just while I'm here.
full
With colors for each alert level (tell me if these are shitty)
 - Green: blarg
 - Blue: blarg
 - Red: blarg
 - Gamma: blarg
 - Epsilon: blarg
 - Delta: blarg

File: nano/styles/_icon.less [hunk 1]
```diff
@@ -11,7 +11,7 @@
     background-image: url(uiLinkPendingIcon.gif);
 }
 
-.linkPending .uiIcon16 {
+.linkPending .fa {
```
Comment (tigercat2000) [inline nano/styles/_icon.less]: this actually means "only affect when `.linkPending` *and* `.fa` are present" (IE, remove icons when showing the linkPending scrollthing)

Comment (tigercat2000) [inline nano/styles/_icon.less]: .. this'll actually make problems for me when trying to change the little spinner thing to fontawesome but whatever, deal with it as I come to it

**Comment by Fox-McCloud** [comment]
security level blue: orange.....what

**Comment by tigercat2000** [comment]
it's the 'average' class, which is used everywhere else as "kinda iffy"

**Comment by Fox-McCloud** [comment]
Any way you could actually make it blue? I just know people are going to complain about that---like um. Switch Gamma and Blue maybe?

**Comment by tigercat2000** [comment]
so picky

**Pull request merged**